rgf/rgf_model.py

```diff
@@ -384,22 +384,24 @@ def _find_model_file(self):
     def dump_model(self):
         """
         Dump forest information to console.
-        ex.
+
+        Examples:
+        ---------
         [  0], depth=0, gain=0.599606, F11, 392.8
           [  1], depth=1, gain=0.818876, F4, 0.6275
             [  3], depth=2, gain=0.806904, F5, 7.226
             [  4], depth=2, gain=0.832003, F4, 0.686
           [  2], (-0.0146), depth=1, gain=0
-        Here, [ x] is order of generated, (x) is weight for leaf nodes, last value is border.
+        Here, [ x] is order of generated, (x) is weight for leaf nodes, last value is a border.
         """
         self._check_fitted()
         cmd = (utils.RGF_PATH, "dump_model", "model_fn=%s" % self._model_file)
         self._execute_command(cmd, verbose=True)
 
     @property
     def feature_importances_(self):
-        """Return the feature importances.
-
+        """
+        The feature importances.
```

Comment:
Here is scikit-learn document.
```python
    @property
    def feature_importances_(self):
        """Return the feature importances (the higher, the more important the
           feature).

        Returns
        -------
        feature_importances_ : array, shape = [n_features]
        """
```

Because `feature_importances_` is a method, to specify "Return" is not bad?

Reply:
It's a property. We don't use "Return" for properties in our package, or am I mistaken?

Reply:
I had no consciousness to unify it.

I think scikit-learn's is more detailed and accurate document, we had better to bring it.

However, unfortunately, rgf other function doesn't specify return type...

Reply:
Yeah, I agree that the current docstrings are not perfect. But I think it's a propose for the future PR. And for the consistency removing "Return" in **this** PR is needed.

Reply:
OK. LGTM